src/core/initializeComponentsFactory.js

```diff
@@ -24,12 +29,20 @@ export default (componentCreators, logger, getNamespacedStorage) => config => {
   config.validate();
   componentCreators.forEach(createComponent => {
     const { namespace } = createComponent;
-    const storage = getNamespacedStorage(config.orgID);
```

Comment:
This doesn't look right. Did you pull the latest changes?

Reply:
I have updated the files.